docs/diagnostics/PX1020.md

```diff
@@ -8,10 +8,12 @@ This document describes the PX1020 diagnostic.
 | PX1020 | For the DAC property with the `AutoNumber` attribute, the string length must be greater than or equal to 15.     | Error | Available   |
 
 ## Diagnostic Description
-You can use the `AutoNumber` attribute on a DAC field to enable auto-numbering for the field. The string must be of the string type and the field length you specify in the `PXDBString` attribute for the field must be at least 15 symbols.
+You can use the `AutoNumber` attribute on a DAC field to enable auto-numbering for the field. For autonumbering to work correctly, the length of the auto-numbered field should not be lesser than the length of a start number of numbering sequence which is specified by the `StartNbr` field of the `PX.Objects.CS.NumberingSequence` DAC.
 
 To fix the issue, you can make the string length sufficient for auto-numbering.
 
+The code fix puts the sufficient length (specified by the `StartNbr` field of the `PX.Objects.CS.NumberingSequence` DAC) as the first parameter value for the `PXDBString` attribute.
```

Comment:
I would rephrase it a little more.  It works also with PXString attribute and should work with attributes derived from PXString/PXDBString.  
It's not very good to rely explicitly on parameters order, so I would just say that the code fix replaces the length value passed to the string attribute's (PXString/PXDBString or derived from them)  constructor with the sufficient length. Which is taken from  `StartNbr` field of the `PX.Objects.CS.NumberingSequence` DAC